src/test/scala/mesosphere/marathon/state/AppRepositoryTest.scala

```diff
@@ -78,7 +78,7 @@ class AppRepositoryTest extends MarathonSpec {
 
     val future = Future.successful(Seq("app1", "app2") ++ allApps.map(x => s"${x.id}:${x.version}"))
 
-    when(store.names()).thenReturn(future)
+    when[Future[Seq[String]]](store.names()).thenReturn(future)
```

Comment:
Hm, I guess it would work without if `val future` would have the type declaration. So `val future: Future[Seq[String]]`. I might be wrong though.